packages/blockfrost/src/blockfrostWalletProvider.ts

```diff
@@ -50,26 +50,18 @@ export const blockfrostWalletProvider = (blockfrost: BlockFrostAPI, logger = dum
     return BlockfrostToCore.blockToTip(block);
   };
 
-  const stakePoolStats: WalletProvider['stakePoolStats'] = async () => {
-    const tallyPools = async (
-      query: 'pools' | 'poolsRetired' | 'poolsRetiring',
-      count = 0,
-      page = 1
-    ): Promise<number> => {
-      const result = await blockfrost[query]({ page });
-      const newCount = count + result.length;
-      if (result.length === 100) {
-        return tallyPools(query, newCount, page + 1);
-      }
-      return newCount;
-    };
-    return {
-      qty: {
-        active: await tallyPools('pools'),
-        retired: await tallyPools('poolsRetired'),
-        retiring: await tallyPools('poolsRetiring')
-      }
-    };
+  const utxoByAddresses: WalletProvider['utxoByAddresses'] = async (addresses) => {
```

Comment:
Why is this being reintroduced? 
https://github.com/input-output-hk/cardano-js-sdk/pull/240/commits/e8142e90997ae2e1a31ea2337d527dbe3fde85b5

Reply:
Removed